usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -447,13 +450,22 @@ def post(self, request, format=None):
                          'treasuryappropriationaccount__account_balances__total_budgetary_resources_amount_cpe'),
                      managing_agency=Subquery(agency_subquery.values('name')[:1]),
                      managing_agency_acronym=Subquery(agency_subquery.values('abbreviation')[:1]))
-
         # add keyword filter, if it exists
         if keyword:
             queryset = queryset.filter(Q(account_name__icontains=keyword) |
                                        Q(account_number__contains=keyword) |
                                        Q(managing_agency__icontains=keyword) |
                                        Q(managing_agency_acronym__contains=keyword.upper()))
+        if agency_id:
+            if agency_id == DOD_CGAC:
```

Comment:
might be cleaner to make this block one line, i.e., 

tta_list = DOD_ARMED_FORCES_CGAC if agency_id == DOD_CGAC else agency_id